### ISSUE TITLE
Update Case Dashboard Accused Field

### ISSUE THREAD

**Issue by jordanngtw** [PublicDataWorks/complaint-manager#259]
### Business Context:

~This story is blocked by #258~

We need to allow the Accused field to show people who are NOT accused officers. 

**As a** person viewing the case dashboard
**I want** to see all of the accused on a case regardless of what person type they are
**So that** I get an accurate picture of who is accused on a case before clicking it

### In Scope
1. Update the `sortable_cases_view` to include civilians and inmates in the list of accused (not just officers, but you need to leave officers so that it still works for NOIPM)
2. Ensure that all accused (including multiple per case) appear on the cases dashboard for each case (should not require front-end changes except maybe renaming `accused_officers` on the sortable_cases_view, just the above change to sortable_cases_view)
3.

### Out of Scope
1.

### Open Questions
1.

## Acceptance Criteria

1. Scenario: Accused people show up on case dashboard
Given I have added an Accused person to a case
When I see that case on the case dashboard
Then the accused person that i added is in the accused column

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)
0. Discuss the technical implementation of this card.
1. Discuss how this feature would be included in or affect the Audit Log.
2. In respect to security, what could go wrong? Brainstorm threats using STRIDE model, especially if this card introduces a new technical architecture or design pattern. (TIMEBOX: 7 mins) https://docs.google.com/document/d/18AFl1d3SRieI4d_8R3GyEazqQOycLNDHGN_vOYIHuiE/edit#heading=h.aio9zb5j63wl
3. Ensure any open questions have been resolved with stakeholders.
4. Determine if documentation should be referenced or created.
5. Ensure to include all necessary mockups to this card.
6. Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
7. Discuss why this feature would not be implemented behind a toggle / what would happen if this card was partially done while a release branch was being cut.
8. If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous  cards in the sequence have been finished.
9. Have we added edge cases as ACs that we're considered from a QA perspective
10. Does this card overlap with any entities outlined in [API Housekeeping](https://www.pivotaltracker.com/story/show/173963545)?  

**Notes:**
